packages/core/src/interpreter.ts

```diff
@@ -142,9 +142,9 @@ export class Interpreter<
         return clearTimeout(id);
       }
     } as Clock,
-    logger: global.console.log.bind(console),
```

Comment:
my rationale here is that the argument passed to this `bind` call wasn't looked up on the `global` variable anyway. This means that all our users were already accessing this in their respective global scopes automatically and it worked. it had to work as otherwise they wouldn't be able to run XState at all as it would throw `Uncaught ReferenceError: console is not defined` at the initialization time.

So I think that we don't have to resolve the global object here manually at all and we can just rely on the standard JS scope lookup.